service-base/src/main/java/org/eclipse/hono/service/tracing/SamplerProducer.java

```diff
@@ -75,46 +132,62 @@ Sampler sampler(final TracerRuntimeConfig tracerRuntimeConfig) {
         return new DropBySpanNamePrefixSampler(sampler, prefixesOfSpansToDrop);
     }
 
-    private static String getProperty(final String name) {
-        return System.getProperty(name, System.getenv(name));
-    }
+    private Sampler getBaseSampler(final String samplerName, final TracerRuntimeConfig.SamplerConfig samplerConfig) {
+        LOG.info("creating OpenTelemetry Sampler [type: {}]", samplerName);
 
-    private static Sampler getBaseSampler(final String samplerName, final Optional<Object> samplerArg) {
-        LOG.info("using OpenTelemetry tracing sampler mode '{}' [arg: {}]", samplerName, samplerArg.orElse(null));
         return switch (samplerName) {
-            case "on" -> Sampler.alwaysOn();
-            case "off" -> Sampler.alwaysOff();
-            case "ratio" -> Sampler.traceIdRatioBased(mapSamplerRatio(samplerArg));
-            case "rate-limiting" -> new RateLimitingSampler(mapSamplerMaxTracesPerSecond(samplerArg));
-            default -> throw new IllegalArgumentException("Unrecognized value for sampler: " + samplerName);
+            // see https://opentelemetry.io/docs/reference/specification/sdk-environment-variables/
+            case "jaeger_remote" -> jaegerRemoteSampler();
+            case "always_on", "on" -> Sampler.alwaysOn();
+            case "always_off", "off" -> Sampler.alwaysOff();
+            case "traceidratio", "ratio" -> traceIdRatioBasedSampler(samplerConfig);
+            // also support rate-limiting sampler directly, i.e. without using Jaeger remote sampler
+            case "rate_limiting", "rate-limiting" -> rateLimitingSampler();
+            default -> Sampler.alwaysOn();
```

Comment:
I think it would be good to have some log output here about the unrecognized config value.